vic/drivers/classic/src/get_global_param.c

```diff
@@ -833,6 +648,29 @@ get_global_param(FILE *gp)
                 log_err("BINARY_STATE_FILE has been replaced with STATE_FORMAT, "
                         "update your global parameter file accordingly");
             }
+            else if (strcasecmp("ALMA_OUTPUT", optstr) == 0) {
+                log_err("ALMA_OUTPUT has been deprecated, update your global "
+                        "parameter file accordingly");
+            }
+            else if (strcasecmp("MOISTFRACT", optstr) == 0) {
+                log_err("MOISTFRACT has been deprecated and has been replaced "
+                        "with two new output variables OUT_SOIL_ICE_FRAC and "
+                        "OUT_SOIL_LIQ_FRAC, update your global parameter file "
+                        "accordingly");
+            }
+            else if (strcasecmp("PRT_HEADER", optstr) == 0) {
+                log_err("PRT_HEADER has been deprecated. All output files "
+                        "include a header including pertinent metadata.");
+            }
+            else if (strcasecmp("PRT_SNOW_BAND", optstr) == 0) {
+                log_err("PRT_SNOW_BAND has been deprecated. To output band "
+                        "specific variables, directly specify them in the "
+                        "global parameter file");
+            }
+            else if (strcasecmp("SKIPYEAR", optstr) == 0) {
+                log_err("SKIPYEAR has been deprecated. To avoid writting output"
```

Comment:
I think the deprecations you are concerned about (`VEGPARAM_VEGCOVER`, and `VEGCOVER_SRC`, `VEGLIB_VEGCOVER`) are all included in the list (c. line 612-626).  You just aren't seeing them since they aren't related to this PR.